### ISSUE TITLE
Linux Support?

### ISSUE THREAD

**Issue by lunarcontrol** [babin101/DevKeyboard#4]
Do you plan to make a linux/Ubuntu version?

**Comment by benwoodman** [comment]
My goal is to eventually push a Linux and OSX version, but right now all I have is a Windows version.

**Comment by LudwikJaniuk** [comment]
I'm not sure if AHK is even on linux

**Comment by benwoodman** [comment]
Yeah it isn't @LudwikJaniuk I would have to develop a whole new script.
